core/src/BitArray.h

```diff
@@ -285,7 +285,7 @@ class BitArray
 	* Clears all bits (sets to false).
 	*/
 	void clearBits() {
-		std::fill(_bits.begin(), _bits.end(), 0);
+        std::fill(_bits.begin(), _bits.end(), uint8_t{});
 	}
```

Comment:
Could you attempt to convince me why this is an improvement (apart from the ws fix)?

Reply:
Just matching contained type and type of the filling value, not faster, more consistent.

Reply:
Added 73ed55c99166d2f3ace630717bc3c41665cca110 to fix the indentation issue. But still ready to withdraw the proposed change.

Reply:
Thanks. I'm not convinced though: The literal constant 0 is definitively more readable and more obvious and the narrowing conversion is 100% correct, always. I would argue that this is actually a false positive warning.

Reply:
OK, reverted in commit 9d2cbaf32f3cfaf4113e27e64740be27e9cc3cf1